modules/ppcp-settings/services.php

```diff
@@ -494,12 +494,14 @@
 		$general_settings = $container->get( 'settings.data.general' );
 		assert( $general_settings instanceof GeneralSettings );
 
+		$acdc = ( $features['advanced_credit_and_debit_cards']['enabled'] ?? false ) && ! $general_settings->own_brand_only();
```

Comment:
It might make sense to use a more descriptive variable name here for clarity.
 ```suggestion
		$is_acdc_enabled = ( $features['advanced_credit_and_debit_cards']['enabled'] ?? false ) && ! $general_settings->own_brand_only();
```

Reply:
Agree @Narek13